NMS-12383: Timeseries Integration Layer (#2845)
OpenNMS/opennms: opennms-doc/guide-admin/src/asciidoc/text/operation/timeseries/introduction.adoc

// Allow GitHub image rendering
:imagesdir: ../../../images

This section describes how to configure _{opennms-product-name}_ to use the _Timeseries Integration Layer_.

Traditionally _{opennms-product-name}_ supports storing timeseries data in _RRD_ files on disk or via _Newts_ in _Cassandra_.
These implementations require a deep knowledge of OpenNMS. It is hard to add another time series database.

With the rise of many new time series databases we want to provide an easy way to support another timseries database with minimal effort.
This lead to the development of the _Timeseries Integration Layer_.

> The  _Timeseries Integration Layer_ allows to integrate a new timeseries database vis osgi plugin.

Reviewer: ```suggestion
The  _Timeseries Integration Layer_ allows new timeseries implementations to be made available via OSGi plugins.
```
